pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -29,9 +28,17 @@ var info = capabilities.MustNewCapabilityInfo(
 	"v1.0.0",
 )
 
+type CapabilityConfig = workflowConfig
```

Comment:
No need for the type alias here I think: just rename `workflowConfig` to `CapabilityConfig`

I think you can simplify the naming btw: CapabilityConfig -> Config; CapabilityInputs -> Inputs, etc